lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,5 +1,10 @@
 require 'rails/all'
+require 'sass-rails'
+require 'coffee-rails'
 require 'patternfly-sass'
+require 'lodash-rails'
+require 'angular-ui-bootstrap-rails'
+require 'jquery-hotkeys-rails'
```

Comment:
Also do these *have* to be required from the start?  I know they were not marked `:require => false` in ManageIQ, but do you know why?